draft-ietf-openpgp-pqc.md

```diff
@@ -321,9 +321,8 @@ Furthermore, multiple OpenPGP signatures may be combined on the application laye
 These latter two cases realize non-composite combinations of signatures.
 {{multiple-signatures}} specifies how implementations should handle the verification of such combinations of signatures.
 
-Furthermore, the OpenPGP protocol also allows for parallel encryption to different keys held by the same recipient.
-Accordingly, if the sender makes use of this feature and sends an encrypted message with multiple PKESK packages for different encryption keys held by the same recipient, a non-composite multi-algorithm public-key encryption is realized where the recipient has to decrypt only one of the PKESK packages in order to decrypt the message.
-See {{no-pq-t-parallel-encryption}} for restrictions on parallel encryption mandated by this specification.
+Furthermore, the OpenPGP protocol also allows for parallel encryption to different keys.
+Accordingly, when a message is encrypted using multiple PKESK packages for different encryption keys, a non-composite multi-algorithm public-key encryption is realized where the communication party has to decrypt only one of the PKESK packages in order to decrypt the message.
```

Comment:
This is not pertaining to the changes, but to the text as was before: Actually, whether the encryption has to be considered "multi algorithm" or not depends on whether the set of keys used belong to different algorithms. So  probably we should rather write: 
```suggestion
Accordingly, when a message is encrypted using multiple PKESK packages for different encryption keys, a non-composite, potentially multi-algorithm, public-key encryption is realized where the communication party has to decrypt only one of the PKESK packages in order to decrypt the message.
```